src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3386,35 +3386,45 @@ components:
     IngestRequestModel:
       required:
         - format
-        - path
         - table
       type: object
       properties:
         table:
           $ref: '#/components/schemas/ObjectNameProperty'
         path:
           type: string
-          description: gs path to a file in a bucket accessible to data repo
+          description: >
+            cloud path path to a file in a bucket accessible to data repo (e.g. gs path for GCP
+            datasets or https path for Azure datasets).  Required if the format is csv or json
```

Comment:
maybe add a comment, do not supply for payload ingest?

Reply:
good call!